challenge_restaurant/src/challenge_restaurant/restaurant.py

```diff
@@ -60,7 +60,7 @@ def __init__(self, robot):
 
             smach.StateMachine.add('NAVIGATE_TO_CUSTOMER',
                                    states.NavigateToObserve(robot=robot, entity_designator=caller_designator,
-                                                            radius=0.85),
+                                                            radius=1.1),
```

Comment:
This is done to increase robustness for SERGIO, since SERGIO is a lot bigger than AMIGO. Although I agree that 1.1 meter might be to far. I suggest to split in half way and make it 1m.